release_notes/ocp-4-7-release-notes.adoc

```diff
@@ -1300,6 +1300,12 @@ The Amazon Web Services (AWS) Elastic File System (EFS) Technology Preview featu
 
 * Technology preview badges were displayed on the OpenShift Pipelines workflows in the web console, though OpenShift Pipelines was GA. The Technology preview badges are now removed. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1945153[*BZ#1945153*])
 
+* Previously, *Pipelines* you created in the git import flow for private repos failed to run. This happened because *Pipeline* ServiceAccount did not use secrets created by git import flow for private git repositories. With this update, you can add a secret name to the pipeline ServiceAccount's annotations, and add pipeline-specific annotations to the provided secret. As a result, *Pipelines* for private git repositories run successfully. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1970485[BZ#1970485])
+
+* Previously, the *Start Modal* did not recognize an empty string as a valid entry, so you had to enter a value. This happened even though the *Tekton Operator* recognizes empty strings as a valid `parameter default`. With this update, you have the option to enter an empty string. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1966275[BZ#1966275])
```

Comment:
```suggestion
* Previously, the *Start Modal* field did not recognize an empty string as a valid entry, so you had to enter a value. This happened even though the *Tekton Operator* recognizes empty strings as a valid `parameter default`. With this update, you have the option to enter an empty string. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1966275[BZ#1966275])
```

Reply:
@rishumehra can you please confirm with @sanketpathak that *Start Modal* and *Tekton Operator* are UI elements? Modal  is commonly used in the developer parlance and not in the docs. Also, I know of OpenShift Pipelines Operator, can you check if Tekton Operator is correct here. Unless it is a GUI element, suggest you remove the bold format for it.

Reply:
Tecton operator is not a UI element but Start Modal is although you will know better regarding how you treat `Modal` word in docs, and I agree with @Preeticp's point to remove bold from  Tecton operator
WDYT @karthikjeeyar

Reply:
Hey @sanketpathak, @karthikjeeyar  is the name of the modal in the UI 'Start Modal' ? We need to capture the name of the dialog box/page as is from the UI.

Reply:
It is 'Start Pipeline Modal'

Reply:
@Preeticp  I just confirmed with @nemesis09 that Start Modal is not a UI element. So we cannot represent it as bold. I will make the changes.

Reply:
@rishumehra please replace Tekton Operator with OpenShift Pipelines Operator